src/utils/UploadStatus.js

```diff
@@ -1,46 +1,24 @@
-class UploadStatus {
-  static UPLOADED = new UploadStatus('uploaded');
-
-  static UPLOADING = new UploadStatus('uploading');
-
-  static UPLOAD_ERROR = new UploadStatus('uploadError');
-
-  static FILE_NOT_FOUND = new UploadStatus('fileNotFound');
-
-  static DATA_MISSING = new UploadStatus('dataMissing');
-
-  static FILE_READ_ERROR = new UploadStatus('fileReadError');
-
-  static FILE_READ_ABORTED = new UploadStatus('fileReadAborted');
-
-  constructor(key) {
-    this.key = key;
-  }
-
-  toString() {
-    return this.key;
-  }
-
-  message() {
-    switch (this) {
-      case UploadStatus.UPLOADED:
-        return 'Uploaded';
-      case UploadStatus.UPLOADING:
-        return 'Uploading...';
-      case UploadStatus.UPLOAD_ERROR:
-        return 'Upload error';
-      case UploadStatus.FILE_NOT_FOUND:
-        return 'File not found';
-      case UploadStatus.DATA_MISSING:
-        return 'Data missing';
-      case UploadStatus.FILE_READ_ERROR:
-        return 'File read error';
-      case UploadStatus.FILE_READ_ABORTED:
-        return 'File read aborted';
-      default:
-        return '';
-    }
-  }
-}
-
+const UploadStatus = {
+  UPLOADED: 'uploaded',
+  UPLOADING: 'uploading',
+  UPLOAD_ERROR: 'uploadError',
+  FILE_NOT_FOUND: 'fileNotFound',
+  DATA_MISSING: 'dataMissing',
+  FILE_READ_ERROR: 'fileReadError',
+  FILE_READ_ABORTED: 'fileReadAborted',
+};
+
+const message = {
+  [UploadStatus.UPLOADED]: 'Uploaded',
+  [UploadStatus.UPLOADING]: 'Uploading...',
+  [UploadStatus.UPLOAD_ERROR]: 'Upload error',
+  [UploadStatus.FILE_NOT_FOUND]: 'File not found',
+  [UploadStatus.DATA_MISSING]: 'Data missing',
+  [UploadStatus.FILE_READ_ERROR]: 'File read error',
+  [UploadStatus.FILE_READ_ABORTED]: 'File read aborted',
+};
+
+const messageFor = (uploadStatus) => message[uploadStatus];
```

Comment:
As this function is specific only for uploadStatus, maybe you can give a more specific name e.g. `messagForStatus`?